### PR TITLE
Adding test for simpack.f

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #26 
Part of #132 

Adding test for simpack. 

I had to link this to the _4 version of the library to get this test to work.

This is a very basic test. I ran some numbers through the function, and printed the output, and wrote a test around it. I still don't have a great understanding what is going on, but this is a start...

### CODE REVIEW

File: src/simpack.f [hunk 1]
```diff
@@ -5,31 +5,34 @@
 !>
 
 !>    This subroutine packs up a data field using a simple packing
-!>    algorithm as defined in the GRIB2 documention. It also fills in GRIB2
-!>    Data Representation Template 5.0 with the appropriate values.
+!>    algorithm as defined in the GRIB2 documention. It also fills in
+!>    GRIB2 Data Representation Template 5.0 with the appropriate
+!>    values.
 !>    
 !>    PROGRAM HISTORY LOG:
```
Comment (BrianCurtis-NOAA) [inline src/simpack.f:12]: Program history log:

**Comment by BrianCurtis-NOAA** [comment]
Opinion request: What do you think about turning the Program history logs into a table?
```
!>    Program history log:
!>    | Date | Author | Comments |
!>    | :--- | :----- | :------- |
!>    |2000-06-21 | Stephen Gilbert | Initial development. |
!>    |2011-10-24 | Boi Vuong | Added variable rmin4 for 4 byte float. |
```
creates
phlv2
:

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Comment by edwardhartnett** [comment]
I think turning them into tables is a good idea and looks great! I will start doing this as I touch code files... Thanks!

**Comment by edwardhartnett** [comment]
OK, I changed to this:

```
!>    ### Program History Log
!>    Date | Author | Comments
!>    -----|--------|--------- 
!>    2000-06-21 | Stephen Gilbert | Initial development.
!>    2011-10-24 | Boi Vuong | Added variable rmin4 for 4 byte float.

```

Which comes out looking nice, like this:

image

**Comment by BrianCurtis-NOAA** [comment]
Yeah that looks even better.

**Pull request merged**